core/src/BibleEngine.class.ts

```diff
@@ -89,7 +89,8 @@ export class BibleEngine {
             currentNormalizedChapter: 0,
             currentNormalizedVerse: -1,
             sectionLevel: 0,
-            recursionLevel: 0
+            recursionLevel: 0,
+            alreadyUsedPhraseIds: {}
```

Comment:
I wonder if this kind of check shouldn't be better placed in a unit test? it uses resources and basically checks for something that shouldn't happen if the code is working (furthermore it doesn't really make sure that the phrase-id isn't used, since it does not check for existing stuff in the DB)